src/linter/ui5Types/fixHints/CoreFixHintsGenerator.ts

```diff
@@ -313,10 +313,40 @@ export default class CoreFixHintsGenerator {
 				exportCodeToBeUsed.args = callExpression.arguments.map((arg) => ({
 					value: arg.getText(),
 					kind: arg?.kind,
+					ui5Type: this.isPotentialUi5Library(arg) ? "library" : undefined,
 				}));
 			}
 		}
 
 		return {...moduleReplacement, exportCodeToBeUsed};
 	}
+
+	isPotentialUi5Library(node: ts.Node): boolean {
+		if (!("text" in node) || typeof node.text !== "string") {
+			return false;
+		}
+
+		const potentialLibNamespace = node.text;
+
+		// Check libs from the manifest
+		const manifest = JSON.parse(this.manifestContent ?? "{}") as SAPJSONSchemaForWebApplicationManifestFile;
```

Comment:
Can't we just re-use the existing logic that extracts manifest.json information instead of doing it again here?
(see https://github.com/SAP/ui5-linter/blob/7b1e6dd11d06c22a200b381ee08a4127431b22de/src/linter/lintWorkspace.ts#L173)

We could change this to a broader scope, like `getInformationFromManifest`, and then returning an object with the relevant details.

Reply:
I just split the existing solution into separate PRs and now I'll do the refactorings

Reply:
Ah, sorry for the premature review 😄